src/main/kotlin/com/cognifide/gradle/aem/instance/LocalInstanceOptions.kt

```diff
@@ -22,7 +22,7 @@ class LocalInstanceOptions(aem: AemExtension) : Serializable {
      *
      * Default path is a subfolder named '.aem' under root project directory.
      */
-    var root: String = aem.props.string("aem.localInstance.root") ?: "${aem.project.rootProject.file(".aem")}"
+    var root: String = aem.props.string("aem.localInstance.root") ?: "${aem.projectMain.file(".aem")}"
```

Comment:
this could affect your existing env created / @mierzwid

Reply:
nope